fleets.json

```diff
@@ -125,7 +137,7 @@
         }
     },
     "meta": {
-        "hostname": "node-01.do-ams3.proxy.misc",
-        "timestamp": "2021-06-04T15:30:41.320046"
+        "hostname": "node-01.do-ams3.sites.misc",
```

Comment:
Any reason why this is changed?

Reply:
That was the server that generated the json in fleets.status.im in that moment